### PR TITLE
Durandal heater support

### PR DESCRIPTION
**Pull request by mhkabir**
Adds support for IMU heater on Durandal.

Additionally, the heater module is greatly simplified.

image

### CODE REVIEW

**Comment by mhkabir** [comment]
We need a better way to auto-detect/select the sensor ID for the heater module to use on supported hardware.

**Comment by mhkabir** [comment]
@davids5 All addressed, thanks for the review!

File: boards/holybro/durandal-v1/init/rc.board_defaults [hunk 1]
```diff
@@ -22,6 +22,9 @@ then
 fi
 unset BL_FILE
 
+# Enable IMU thermal control
+param set SENS_EN_THERMAL 1
```
Comment (mhkabir) [inline boards/holybro/durandal-v1/init/rc.board_defaults]: May want to make this an AUTOCNF thing to give users the flexibility to disable heater functionality. Right now, this force-enables it every boot.

Thoughts @dagar?

Comment (dagar) [reply boards/holybro/durandal-v1/init/rc.board_defaults]: Ha, I was just about to suggest that.

File: src/drivers/heater/heater.cpp [hunk 2]
```diff
@@ -88,35 +72,7 @@ int Heater::custom_command(int argc, char *argv[])
...
-	}
-
-	// Displays/Set the current IMU temperature setpoint.
-	if (strcmp(arg_v, "setpoint") == 0) {
-		return get_instance()->temperature_setpoint(argv);
-	}
-
-	get_instance()->print_usage("Unrecognized command.");
-	return PX4_OK;
+	return print_usage("Unrecognized command.");
```
Comment (dakejahl) [inline src/drivers/heater/heater.cpp]: Is the argument for removing these thaat to adjust gains you need to let the board cool anyways so you may as well reboot?

**Comment by dakejahl** [comment]
Looks good to me.

File: boards/holybro/durandal-v1/init/rc.board_defaults (hunk 1 above)
Comment (davids5) [reply boards/holybro/durandal-v1/init/rc.board_defaults]: Is there an update coming?

Comment (davids5) [reply boards/holybro/durandal-v1/init/rc.board_defaults]: Is there a change coming?

**Comment by mhkabir** [comment]
@davids5 Done. Good to merge once CI is green.

I'll make a follow-up for sensor ID autodetection.

**Comment by mhkabir** [comment]
CI fail unrelated to changes.

**Pull request merged**